packages/ng/src/app/select/option/select-option-feeder.model.ts

```diff
@@ -0,0 +1,25 @@
+export interface ISelectOptionFeeder<T> {
+
+	/**
+	 * @returns true if the focus is on the field
+	 */
+	hasFocus(): boolean;
```

Comment:
wot ? why not an `Observable<boolean>` ? you can make it a `BehaviourSubject` if you want access to the last emitted value

Reply:
Observable for this is overkill the idea behind this method is to told to the directive (the trigger of popover) that the focus is still controlled by an element of select => a ISelectOptionFeeder. But only the ISelectOption knows if he has the focus. For example, with the select-searcher component, the focus is true when the user click on the input that filter the elements.

Having an observable mecanism here will be to complicated. HasFocus is just call when an onBlur is detect